templates/overlappingWithModule.html

```diff
@@ -0,0 +1,37 @@
+$def with (moduleCode, overlappingList)
```

Comment:
If the table is supposed to only show the overlapping modules for a **target AY-Sem** only, then can remove the AY-Sem column. And should indicate the AY-Sem in the header too.

But currently, this overlapping page can be accessed from both the module overview and the module view in ay sem. So I'm not sure if it is supposed to filter only for a target AY-Sem. If it is, the nav button should be removed from module overview.